app/controllers/evidence_controller.rb

```diff
@@ -6,11 +6,13 @@ class EvidenceController < NestedNodeResourceController
   include MultipleDestroy
   include NodesSidebar
   include NotificationsReader
+  include Previewable
 
-  before_action :set_or_initialize_evidence, except: [ :index, :create_multiple ]
+  before_action :set_or_initialize_evidence, except: [ :index, :create_multiple, :preview ]
```

Comment:
Can we update this to use `only:` ? That way we're using a safer approach (whitelist) and it's more consistent with the rest of the callbacks below.